src/module/canvas/status-effects.ts

```diff
@@ -274,6 +274,7 @@ export class StatusEffects {
      * @param event The window click event
      */
     static async #setStatusValue(token: TokenPF2e, event: JQuery.TriggeredEvent): Promise<void> {
+        event.preventDefault();
```

Comment:
There is no default default action here.

Reply:
It could be moved inside the `contextmenu` check that happens a bit later in the method, but the default behaviour needs to be prevented for right clicks.

Reply:
That's the thing, though: there is no default action in this case.

Reply:
![Recording 2022-08-14 at 17 43 07](https://user-images.githubusercontent.com/41452412/184544591-186e501e-9924-49cb-9b6d-48f7efaaf976.gif)

Reply:
I'm not getting that here. :\

Reply:
Chrome 104.0.5112.81 on Windows 10